src/PhotosphereEditor.tsx

```diff
@@ -251,6 +269,8 @@ function PhotosphereEditor({
           onAddHotspot={handleAddHotspot}
           pitch={pitch}
           yaw={yaw}
+          photosphereOptions={availablePhotosphereOptions}
+          currentPS={currentPS}
```

Comment:
I don't think this prop is needed anymore since the current photosphere is already filtered out above.